### DISCUSSION TITLE
How do we customize the entire Friday Night Funkin

### DISCUSSION THREAD

**Discussion by Dark2481** [KadeArchive/Kade-Engine#828] [Q&A]
This is a hard question some people knows how they can customize the entire Friday night funkin but I don't know how it might be hard to think

**Comment by myst-rblx** [comment]
huhhhhhhhhhhhhhh

**Comment by Lucky-56** [comment]
hmmmmmmm....
\
\
\
\
\
(I don't know if this is a joke, but please tell me it is)
